### PR TITLE
feat: add dashboard playground link card

### LINKED ISSUE #667: feat: add Apache APISIX dashboard playground on website
**Actual Behaviour**

Apache APISIX dashboard demo shows on apache/apisix-dashboard readme, but it is so small. Sometimes I can't find it. 

**Expected Behaviour**

I hope to add a dashboard entrance to the website.

**Screenshots of the issue**

image

### PR DESCRIPTION
**Pull request by SkyeYoung**
Fixes: #667

Changes:

- reformat code
- add dashboard playground block

Screenshots of the change:

Snipaste_2022-03-22_16-46-43

dashboard-playground-2

### CODE REVIEW

**Comment by SkyeYoung** [comment]
@juzhiyuan pls have a look of the screenshot. It's hard to design it. Do you have any suggestions?😅

**Comment by SkyeYoung** [comment]
> Build preview is broken since yesterday. Any idea why? CDN issue?

It seems that there is something wrong with CDN. I will seek help from others to repair this problem. Now you may have to pull this branch locally for preview. So Sorry.

**Comment by juzhiyuan** [comment]
> > Build preview is broken since yesterday. Any idea why? CDN issue?
> 
> It seems that there is something wrong with CDN. I will seek help from others to repair this problem. Now you may have to pull this branch locally for preview. So Sorry.

This is not your fault, due to some reasons, jsDelivry doesn't work well in China. I'm planning a PR to use CDN from github raw.

**Comment by SkyeYoung** [comment]
@1502shivam-singh I updated it, pls review this PR. I have already burn out. :D

**Review by bzp2010 — APPROVED** [review]
LGTM

**Comment by Baoyuantop** [comment]
How about replacing PASSWD with PASSWORD?🤔

**Comment by SkyeYoung** [comment]
> How about replacing PASSWD with PASSWORD?🤔

Look [here](https://github.com/apache/apisix-website/issues/667#issuecomment-1073233179)

File: website/src/css/customTheme.css [hunk 1]
```diff
@@ -53,45 +53,60 @@ html[data-theme="dark"] {
 }
 
 @font-face {
-  font-family: 'MaisonNeue-Medium';
-  src: local('MaisonNeue-Medium'), url(https://raw.githubusercontent.com/apache/apisix-website/master/website/src/fonts/MaisonNeue-Medium.otf) format('opentype');
+  font-family: "MaisonNeue-Medium";
+  src: local("MaisonNeue-Medium"),
+  url(https://cdn.jsdelivr.net/gh/apache/apisix-website@master/website/src/fonts/MaisonNeue-Medium.otf) format("opentype");
```
Comment (juzhiyuan) [inline website/src/css/customTheme.css:58]: @bzp2010 jsDelivr will return 4xx sometimes, is it ok to still use it?

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**